src/_components/StudentsTeams.jsx

```diff
@@ -0,0 +1,12 @@
+export default function StudentsTeams({ comp }) {
```

Comment:
I think you could probably remove this component and just place the `<h1>` and `<section>` tag into the `<StudentsProjectTeam>`  component.